data/presets/climbing/route_top.json

```diff
@@ -0,0 +1,11 @@
+{
+    "icon": "temaki-climbing",
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "tags": {
+        "climbing": "route_top"
```

Comment:
Isn't the top always the end of the way? It does not make sense to have a top node without any way or relation between the start, right? And once there is a way, the top node does not add to the data… (except for the fact that it makes explicit that there is no `fixme=continue`) on the way.

Are there any additional fields that this preset would get?

I wonder if we should make it an unsearchable preset (renamte to `_foo.json` and add [`searchable:false`](https://github.com/ideditor/schema-builder?tab=readme-ov-file#searchable)), so it gets rendered in the UI and shows something when clicked… but it is not promoted via the search and category list.